change/@fluentui-react-charting-4c197bf1-3dca-480d-aa17-d97a170246ff.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Enabled Multi Select behaviour of Controlled legends",
```

Comment:
>Controlled [](http://example.com/codeflow?start=48&length=10)

Make corresponding change in declarative chart and fluent_line_chart.json schema example.

Reply:
done and tested.